lib/graphiti/util/persistence.rb

```diff
@@ -180,12 +164,8 @@ def process_has_many(relationships, caller_model)
       iterate(except: [:polymorphic_belongs_to, :belongs_to]) do |x|
         yield x
 
-        if x[:sideload].writable?
```

Comment:
because this check happens upstream in the validator right?

Reply:
As far as I can tell this can't happen in a way that doesn't go through the runner which does the validator check. Let me know if this isn't true for any reason you're aware of.